app/test_main.py

```diff
@@ -1,3 +1,19 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_get_human_age() -> None:
+    assert get_human_age(0, 0) == [0, 0]
+
+    assert get_human_age(14, 14) == [0, 0]
+
+    assert get_human_age(15, 15) == [1, 1]
+
+    assert get_human_age(23, 23) == [1, 1]
+
+    assert get_human_age(24, 24) == [2, 2]
+
+    assert get_human_age(27, 27) == [2, 2]
+
+    assert get_human_age(28, 28) == [3, 2]
+
+    assert get_human_age(100, 100) == [21, 17]
```

Comment:
have you read the [checklist](https://github.com/mate-academy/py-cat-and-dog-years/blob/master/checklist.md)?????

<img width="909" alt="image" src="https://github.com/mate-academy/py-cat-and-dog-years/assets/93856423/b5c3b1f8-0cba-4e78-92b8-90489c127d3a">